subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -763,8 +771,8 @@ config BT_CTLR_SCHED_ADVANCED
 	depends on BT_CTLR_SCHED_ADVANCED_SUPPORT && \
 		   (BT_CONN || \
 		    (BT_CTLR_ADV_EXT && (BT_CTLR_ADV_AUX_SET > 0)) || \
-		    BT_CTLR_ADV_ISO)
-	select BT_TICKER_NEXT_SLOT_GET
+		    BT_CTLR_ADV_ISO) && \
+		   BT_TICKER_NEXT_SLOT_GET
```

Comment:
Both the rename and the change to depends on may cause existing builds to fail. I suggest to make an entry in the migration guide to explain how to handle such build errors